packages/backend/src/accounting/service.ts

```diff
@@ -183,6 +195,21 @@ export async function createAccountToAccountTransfer(
           withdrawalThrottleDelay
         })
       }
+      if (
+        destinationAccount.onDebit &&
+        telemetry &&
+        aseRatesService &&
+        sourceAccount.asset.code &&
+        sourceAccount.asset.scale
+      ) {
+        collectTelemetryAmount(telemetry, aseRatesService, {
```

Comment:
should this be awaited?

Reply:
Or I'm guessing you don't need to await since we don't use the response of this function? 
It makes sense, I'm just always wary of not `await`ing an async fn mostly because of error handling - I don't think an error can be caught from this method - (I think) it would always throw an unhandled promise rejection.

Reply:
not awaiting is a choice, i'd rather have the accounting execution keep flowing without it bothering about telemetry. 
It gets executed in the background, as telemetry should be.

in terms of error handling, you are correct.
The function itself will just catch whatever goes wrong inside it, and just log.
Does that sound ok to you?